### ISSUE TITLE
add option to send headers/cookies

### ISSUE THREAD

**Issue by cottsak** [Y2Z/monolith#9]
It would be great to use this as a final step in a selenium scrape or other browser driver which may require session state. Then this could pick up the final step (session included) and dump the desired result.

**Comment by snshn** [comment]
Related to #55 , it's now implemented and in `master`, will be in the next release (flag `-C` for providing cookie file name, specifying charset has been moved to `-E`).

**Comment by snshn** [comment]
It is now in 2.8.0.

Please feel free to close this issue if it satisfies the originally desired functionality.

**Comment by snshn** [comment]
Good point.  The tricky part is that we don't want to leak those credentials by sending them everywhere (with every network request).  The cookie file is safe, since it contains domain/path information in it, but the Authentication header needs to have a target.  I could make Monolith send it only to the hostname of the target, but then if the target URL redirects (e.g. from something.com to www.something.com), it wouldn't work.

**Comment by snshn** [comment]
Lemme see if `http://username:password@example.com/` works.
